pkg/web_css/lib/src/_list.scss

```diff
@@ -80,6 +80,55 @@
   }
 }
 
+.modal-carousel {
```

Comment:
Could these (or a subset of these) styles be embedded under a top-level style, e.g. `.carousel`? It is not really needed for anything, but I like the aspect of grouping both visually here, and also as a "namespace" in CSS.

Reply:
cone